packages/x-data-grid/src/models/props/DataGridProps.ts

```diff
@@ -417,6 +417,10 @@ export interface DataGridPropsWithoutDefaultValue<R extends GridValidRowModel =
    * Override the height/width of the Data Grid inner scrollbar.
    */
   scrollbarSize?: number;
+  /**
+   * Override the height/width of the header filters.
```

Comment:
I don't understand how does it impact the width
```suggestion
   * Sets the height in pixel of the column headers filters row in the Data Grid.
   * @default `props.columnHeaderHeight`
```